tests/e2e/specs/admin/amp-options.js

```diff
@@ -113,7 +113,7 @@ describe( 'Saving', () => {
 	it( 'allows saving', async () => {
 		const testSave = async () => {
 			await expect( page ).toClick( 'button', { text: 'Save' } );
-			await page.waitForSelector( 'button[disabled].is-busy' );
+			await page.waitForSelector( 'button[disabled].is-busy', { timeout: 1000 } );
```

Comment:
@westonruter This can be reverted now that it's being fixed via https://github.com/ampproject/amp-wp/pull/6613.